vacs-map-app/src/components/layouts/LayoutOverview.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="layout-overview">
+    <OverviewTop :topic="topic" />
+    <div class="layout-overview-content">
+      <slot></slot>
+    </div>
+  </div>
+</template>
```

Comment:
The idea would be that we would use these layout components to handle common layout concerns.